### PR TITLE
Add About Us, Update Refute UIS Response and Change Title of Section to Respond to Pushback

### LINKED ISSUE #202: Change Refute UIS Response to Responding to Pushback and Update Style
## Description

Rename the Refute UIS Response section to Responding to Pushback and implement small style changes to the Card Carousel. 

Image

## Todo

- [x] Rename section to Responding to Pushback
- [x] Update contentful to accommodate style changes
- [x] Implement claim/response style changes

### LINKED ISSUE #203: About Us Section
## Description

The About Us section will go in between Responding to Pushback (Issue #202) and the Timeline Section

Image

## Todo

- [x] Add About Us Section
- [x] Add Text to Contentful

### PR DESCRIPTION
**Pull request by kennykos**
## Summary

Closes #203 , Closes #202 

## Changes

- Updated README.md

- Add about us section
- Remove sample section
- Fix old spelling bugs
- Change background on timeline to reflect background color pattern

- Rename Refute UIS Response to  Respond to Pushback
- Implement new contentful model for responses
- implement new render stying for responses
- Add subheader-1 with graph ff condensed black font

## Screenshots

image
image

### CODE REVIEW

File: src/sections/home/RefuteUISResponse.tsx [hunk 1]
```diff
@@ -15,13 +18,14 @@ const RefuteUISResponse: FC<RefuteUISResponseProps> = ({
   leftText,
   entries,
 }) => {
-  const carouselChildren = entries.map((entry) =>
-    documentToReactComponents(entry.details, mainParagraphStyle),
-  )
+  const carouselChildren = entries.map((entry) => [
+    documentToReactComponents(entry.claim, claimParagraphStyle),
+    documentToReactComponents(entry.response, responseParagraphStyle),
```
Comment (zorazrr) [inline src/sections/home/RefuteUISResponse.tsx:23]: In `renderer.tsx` there is a decent amount of overlapping code in `claimParagraphStyle`, `responseParagraphStyle` and `mainParagraphStyle`. You could probably directly specify the "Claim" and "Response" header here in this map function?

**Review by zorazrr — COMMENTED** [review]
Some small nits.

File: src/sections/home/RefuteUISResponse.tsx (hunk 1 above)
Comment (kennykos) [reply src/sections/home/RefuteUISResponse.tsx]: Addressed in commit [39ddd99335e5174c0f15ab0e47c614ab7c01038b](https://github.com/toriis-portal/toriis/pull/204/commits/39ddd99335e5174c0f15ab0e47c614ab7c01038b)

**Review by zorazrr — APPROVED** [review]
LGTM!

**Pull request merged**